object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/FrameworkSupport.java

```diff
@@ -0,0 +1,38 @@
+package org.checkerframework.checker.objectconstruction.framework;
+
+import com.sun.source.tree.NewClassTree;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+
+/** add CalledMethods annotations to code generated by framework */
+public interface FrameworkSupport {
+
+  /**
+   * Allows for supporting a framework's {@code toBuilder} routine. Typically, the returned Builder
+   * setters invoked. So, the framework support can add a {@link
```

Comment:
"Typically, the returned Builder setters invoked." doesn't parse in English. I think a verb is needed, or maybe more...I might rewrite as: "Typically, the returned Builder *has had some* setters invoked."

Reply:
ping